build/common.project.props

```diff
@@ -271,4 +271,8 @@
    <ApexProjects Include="$(RepositoryRootDirectory)test\NuGet.Tests.Apex\*\*.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
I'm not a fan of having this package in every single one of our libraries. 

Especially not as dependency. 

This is a build time dependency, so it should be marked with PrivateAssets="all". 
It should also be conditioned on the tfm as it's only relevant for the .NET framework builds. 

Ideally I'd like to avoid having a copy of the package declaration in every csproj though, but I'm not sure if there's a better way until we start using Directory.Build.props everywhere. 

//cc @NuGet/nuget-client thoughts?

Reply:
Agreed. What is this package? I cant find it on NuGet Gallery.

If this is needed as a build tool then we should add it with PdbGit i.e. [here](https://github.com/NuGet/NuGet.Client/blob/dev/build/common.props#L8) and [here](https://github.com/NuGet/NuGet.Client/blob/dev/build/common.legacy.props#L7).

Reply:
Updated with PrivateAssets and condition, I can try referencing the package individually in csprojs if that would be better.

Reply:
Sorry, didn't get this update before I pushed the change.  Looking at this approach now.

Reply:
Moved to `common.props`, I don't think the projects using `common.legacy.props` are built in our configuration so I did not add it there.